src/glob.rs

```diff
@@ -7,6 +7,7 @@ use crate::settings::Settings;
 pub fn glob_exec<F: FnMut(&Path)>(base: &Path, pattern: &str, mut f: F) {
     let walker = GlobWalkerBuilder::new(base, pattern)
         .case_insensitive(true)
+        .follow_links(true)
         .file_type(FileType::FILE)
```

Comment:
Tangentially, what's the reason for constraining globs to only match regular files?

I have a project where my inputs for tests are directories, not files, and it would be handy to be able to use the `glob!` macro (right now I loop through them myself).